requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
```

Comment:
Won't be using the standard python `requests` library with FastAPI as requests is a blocking http client. Instances of requests should be replaced with the more async-native `aiohttp` library.

Reply:
'requests' is used by TestClient

Reply:
Ahh ok. In that case, `requests` should go into requirements-dev.txt, that way the application isn't pulling in extra libraries at runtime that aren't actual runtime dependencies.